pkg/invoke/raw_exec.go

```diff
@@ -31,34 +31,40 @@ type RawExec struct {
 
 func (e *RawExec) ExecPlugin(ctx context.Context, pluginPath string, stdinData []byte, environ []string) ([]byte, error) {
 	stdout := &bytes.Buffer{}
+	stderr := &bytes.Buffer{}
 	c := exec.CommandContext(ctx, pluginPath)
 	c.Env = environ
 	c.Stdin = bytes.NewBuffer(stdinData)
 	c.Stdout = stdout
-	c.Stderr = e.Stderr
+	c.Stderr = stderr
 	if err := c.Run(); err != nil {
-		return nil, pluginErr(err, stdout.Bytes())
+		return nil, e.pluginErr(err, stdout.Bytes(), stderr.Bytes())
 	}
 
+	// Copy stderr to caller's buffer in case plugin printed to both
+	// stdout and stderr for some reason
+	if e.Stderr != nil && stderr.Len() > 0 {
+		if _, err := stderr.WriteTo(e.Stderr); err != nil {
```

Comment:
I'd be okay with ignoring this error. Stderr is informational.

Reply:
Done.